src/Control/Email/Swift/Transport/MailTransport.php

```diff
@@ -0,0 +1,315 @@
+<?php
+
+/**
+ * This file was copied in from swiftmailer/swiftmailer v5.4.12 after it was removed from switftmailer v6
+ * It has been slightly modified to meet phpcs standards and to update method signatures to match the swiftmailer v6
+ */
+
+/*
+ * This file is part of SwiftMailer.
+ * (c) 2004-2009 Chris Corbyn
+ *
+ * For the full copyright and license information, please view the LICENSE file (MIT)
+ * https://github.com/swiftmailer/swiftmailer/blob/181b89f18a90f8925ef805f950d47a7190e9b950/LICENSE
+ */
+
+/**
+ * Sends Messages using the mail() function.
+ *
+ * It is advised that users do not use this transport if at all possible
+ * since a number of plugin features cannot be used in conjunction with this
+ * transport due to the internal interface in PHP itself.
+ *
+ * The level of error reporting with this transport is incredibly weak, again
+ * due to limitations of PHP's internal mail() function.  You'll get an
+ * all-or-nothing result from sending.
+ *
+ * @author Chris Corbyn
+ *
+ * at deprecated since 5.4.5 (to be removed in 6.0)
+ *
+ * @internal
+ */
+// @codingStandardsIgnoreStart
+// ignore missing namespace
+class Swift_Transport_MailTransport implements Swift_Transport
+// @codingStandardsIgnoreEnd
+{
+    /** Additional parameters to pass to mail() */
+    private $_extraParams = '-f%s';
+
+    /** The event dispatcher from the plugin API */
+    private $_eventDispatcher;
+
+    /** An invoker that calls the mail() function */
+    private $_invoker;
+
+    /**
+     * Create a new MailTransport with the $log.
+     *
+     * @param Swift_Transport_MailInvoker  $invoker
+     * @param Swift_Events_EventDispatcher $eventDispatcher
+     */
+    public function __construct(Swift_Transport_MailInvoker $invoker, Swift_Events_EventDispatcher $eventDispatcher)
+    {
+        @trigger_error(sprintf('The %s class is deprecated since version 5.4.5 and will be removed in 6.0. Use the Sendmail or SMTP transport instead.', __CLASS__), E_USER_DEPRECATED);
```

Comment:
This deprecation can be removed. We may wish to add our **own** deprecation notice relevant to SS, though?

Reply:
I've removed the switftmailer deprecated notice.  We don't need to add a deprecation notice since we're not deprecating it, in fact we're basically doing the opposite :-)